doc/sphinx/source/contributing.rst

```diff
@@ -25,6 +25,11 @@ repository, each commit should respect a few rules:
   of the commit log message - it certifies that the committer has the rights to
   submit the work under the project's license.
 
+- The commit message should follow the **50/72 rule** :
```

Comment:
AFAIK this is more of a recommendation than a hard rule even in linux development.